### PR TITLE
github(README): update dev team

### PR DESCRIPTION
**Pull request by adr-mo**
Simply update dev team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,22 +38,20 @@ on the market; it is absolutely free and Open Souce.
...
 <a href="https://github.com/Pontissalien"><img src="https://avatars3.githubusercontent.com/u/38663853?s=400&v=4" title="Thomas Arnaud" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/agelibert"><img src="https://avatars1.githubusercontent.com/u/28709386?s=400&v=4" title="Adrien Gélibert" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/Thebarda"><img src="https://avatars3.githubusercontent.com/u/12515407?s=400&u=5642f34672a7c1ea887c6c3e12cd28880aa3971e&v=4" title="Tom Darneix" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/bdauria"><img src="https://avatars0.githubusercontent.com/u/8367233?s=400&u=563bd982e8d1557bdfd39627b029610d9c83fdc6&v=4" title="Bruno D'Auria" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/adr-mo "><img src="https://avatars2.githubusercontent.com/u/31647811?s=400&v=4" title="Adrien Morais" width="80" height="80"></a> &nbsp;
+<a href="https://github.com/jeremyjaouen "><img src="https://avatars.githubusercontent.com/u/61694165?s=400&v=4" title="Jérémy Jaouen" width="80" height="80"></a> &nbsp;
 
 <h4> Quality Assurance </h2>
```
Comment (sc979) [inline README.md:57]: Could you also add Charles ?
https://github.com/chgautier

File: README.md [hunk 2]
```diff
@@ -38,22 +38,20 @@ on the market; it is absolutely free and Open Souce.
...
 <a href="https://github.com/Pontissalien"><img src="https://avatars3.githubusercontent.com/u/38663853?s=400&v=4" title="Thomas Arnaud" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/agelibert"><img src="https://avatars1.githubusercontent.com/u/28709386?s=400&v=4" title="Adrien Gélibert" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/Thebarda"><img src="https://avatars3.githubusercontent.com/u/12515407?s=400&u=5642f34672a7c1ea887c6c3e12cd28880aa3971e&v=4" title="Tom Darneix" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/bdauria"><img src="https://avatars0.githubusercontent.com/u/8367233?s=400&u=563bd982e8d1557bdfd39627b029610d9c83fdc6&v=4" title="Bruno D'Auria" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/adr-mo "><img src="https://avatars2.githubusercontent.com/u/31647811?s=400&v=4" title="Adrien Morais" width="80" height="80"></a> &nbsp;
+<a href="https://github.com/jeremyjaouen "><img src="https://avatars.githubusercontent.com/u/61694165?s=400&v=4" title="Jérémy Jaouen" width="80" height="80"></a> &nbsp;
```
Comment (sc979) [inline README.md:54]: Can we also add him to the Centreon About pages ?

Comment (sc979) [reply README.md:55]: done here https://github.com/centreon/centreon/pull/9521

File: README.md (hunk 1 above)
Comment (sc979) [reply README.md:58]: added

File: README.md [hunk 3]
```diff
@@ -38,27 +39,26 @@ on the market; it is absolutely free and Open Souce.
...
-<a href="https://github.com/vhr"><img src="https://avatars2.githubusercontent.com/u/365020?s=400&v=4" title="Valentin Hristov" width="80" height="80"></a> &nbsp;
-<a href="https://github.com/ganoze"><img src="https://avatars0.githubusercontent.com/u/6575378?s=400&v=4" title="Matthieu Kermagoret" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/loiclau"><img src="https://avatars3.githubusercontent.com/u/2759315?s=400&v=4" title="Loic Laurent" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/leoncx"><img src="https://avatars1.githubusercontent.com/u/3930276?s=400&v=4" title="Maximilien Bersoult" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/jiliangWANG"><img src="https://avatars1.githubusercontent.com/u/48441685?s=400&v=4" title="Jiliang Wang" width="80" height="80"></a> &nbsp;
-<a href="https://github.com/SylvestreG"><img src="https://avatars2.githubusercontent.com/u/3818949?s=400&v=4" title="Sylvestre Gallon" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/Pontissalien"><img src="https://avatars3.githubusercontent.com/u/38663853?s=400&v=4" title="Thomas Arnaud" width="80" height="80"></a> &nbsp;
```
Comment (Nohzoh) [inline README.md:50]:
```suggestion
<a href="https://github.com/Nohzoh"><img src="https://avatars3.githubusercontent.com/u/38663853?s=400&v=4" title="Thomas Arnaud" width="80" height="80"></a> &nbsp;
```

**Review by Nohzoh — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (adr-mo) [reply README.md]: thanks 💯

**Review by sc979 — APPROVED** [review]

**Review by Nohzoh — APPROVED** [review]

**Pull request merged**